### ISSUE TITLE
Where the issues about bugs on the weather service, a double tap feature and the pong game and went?

### ISSUE THREAD

**Issue by tokiwee** [InfiniTimeOrg/InfiniTime#2189]
Where the issues about bugs on the weather service, a double tap feature and the pong game and disappeared??

**Comment by mark9064** [comment]
I can assure you that no issues have been deleted or otherwise removed by the InfiniTime team. If you have not deleted them yourself, you will need to get in touch with GitHub support

**Comment by vkareh** [comment]
yeah that's weird... I definitely have those threads in my email:
![Screenshot at 2024-12-09 16-58-51](https://github.com/user-attachments/assets/6f4c1b75-28d9-4dd7-a39a-3b94a7b360a3)

but trying to get there I get a 404:
![Screenshot at 2024-12-09 16-57-53](https://github.com/user-attachments/assets/46c6f06c-8a30-4770-bdb0-ab8cbb321f0d)

**Comment by JF002** [comment]
As @mark9064 said, we haven't removed any of those issues. Github might have an anti-spam feature that removed them for some reasons?
Anyway, I suggest you contact Github support about this.
Feel also free to create them (maybe just one to begin with) to see what happens.
